### PR TITLE
[R2] Add an Overview page

### PR DESCRIPTION
**Pull request by KianNH**
Mostly just a renamed https://developers.cloudflare.com/workers/ really - but I think it'd be nice to have a homepage for R2

### CODE REVIEW

**Comment by KianNH** [comment]
https://patch-53.cloudflare-docs-github.pages.dev/r2/

File: content/r2/_index.md [hunk 1]
```diff
@@ -12,5 +12,27 @@ meta:
...
+{{</button-group>}}
 
-{{</content-column>}}
\ No newline at end of file
+---
+
+## Popular pages
+
+- [Pricing](/r2/platform/pricing/) – Learn about the free tier and paid rates.
+- [Reference: Changelog](/r2/platform/changelog/) – Keep up-to-date with what's new in R2.
```
Comment (deadlypants1973) [inline content/r2/_index.md]:
```suggestion
- [Changelog](/r2/platform/changelog/) – Review updates in R2.
```

File: content/r2/_index.md [hunk 2]
```diff
@@ -12,5 +12,27 @@ meta:
...
+{{<button type="primary" href="/r2/get-started/">}}Get started{{</button>}}
+{{<button type="secondary" href="/r2/examples/">}}View the examples{{</button>}}
+{{<button type="secondary" href="/r2/runtime-apis/">}}View the Workers APIs{{</button>}}
+{{</button-group>}}
 
-{{</content-column>}}
\ No newline at end of file
+---
+
+## Popular pages
```
Comment (deadlypants1973) [inline content/r2/_index.md]:
```suggestion
## Related Resources
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/r2/_index.md [hunk 3]
```diff
@@ -12,5 +12,27 @@ meta:
 # Cloudflare R2 documentation
 
 Cloudflare R2 Storage allows developers to store large amounts of unstructured data without the costly egress bandwidth fees associated with typical cloud storage services.
+  
+{{<button-group>}}
+{{<button type="primary" href="/r2/get-started/">}}Get started{{</button>}}
+{{<button type="secondary" href="/r2/examples/">}}View the examples{{</button>}}
```
Comment (deadlypants1973) [inline content/r2/_index.md]:
```suggestion
{{<button type="secondary" href="/r2/examples/">}}Browse the examples{{</button>}}
```

File: content/r2/_index.md [hunk 4]
```diff
@@ -12,5 +12,27 @@ meta:
...
+{{<button type="secondary" href="/r2/runtime-apis/">}}View the Workers APIs{{</button>}}
```
Comment (deadlypants1973) [inline content/r2/_index.md]:
```suggestion
{{<button type="secondary" href="/r2/runtime-apis/">}}Review the Workers APIs{{</button>}}
```

**Review by deadlypants1973 — APPROVED** [review]

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**